docs_user/modules/proc_deploying-the-bare-metal-provisioning-service.adoc

```diff
@@ -94,32 +91,36 @@ spec:
       secret: osp-secret
 '
 ----
+
+. Wait for the {bare_metal} control plane services CRs to become ready:
 +
-The operator begins to apply the configuration and start the necessary Bare Metal Provisioning services. Once the services have reached a running state, the {bare_metal} automatically begins polling the power state of bare metal nodes for which it is configured to manage.
+----
+$ oc wait --for condition=Ready --timeout=300s ironics.ironic.openstack.org ironic
+----
 
-. Wait for Bare Metal Provisioning control plane services' custom resources to become ready:
+. Optional: Verify that the individual services are ready:
 +
 ----
-oc wait --for condition=Ready --timeout=300s ironics.ironic.openstack.org ironic
-//kgilliga: Is "optionally verify the individual service" part of the code block, or is it a separate step?
-# Optionally verify the individual services
-oc wait --for condition=Ready --timeout=300s ironicapis.ironic.openstack.org ironic-api
-oc wait --for condition=Ready --timeout=300s ironicconductors.ironic.openstack.org ironic-conductor
-oc wait --for condition=Ready --timeout=300s ironicinspectors.ironic.openstack.org ironic-inspector
-oc wait --for condition=Ready --timeout=300s ironicneutronagents.ironic.openstack.org ironic-ironic-neutron-agent
+$ oc wait --for condition=Ready --timeout=300s ironicapis.ironic.openstack.org ironic-api
+$ oc wait --for condition=Ready --timeout=300s ironicconductors.ironic.openstack.org ironic-conductor
+$ oc wait --for condition=Ready --timeout=300s ironicinspectors.ironic.openstack.org ironic-inspector
+$ oc wait --for condition=Ready --timeout=300s ironicneutronagents.ironic.openstack.org ironic-ironic-neutron-agent
 ----
 
-. Update the DNS Nameservers on the provisoning/cleaning/rescue networks. For name resolution to work for {bare_metal} operations the DNS nameserver must be set to use the internal DNS servers in the {rhos_acro} control plane:
+. Update the DNS Nameservers on the provisoning, cleaning, and rescue networks:
++
+[NOTE]
+For name resolution to work for {bare_metal} operations, you must set the DNS nameserver to use the internal DNS servers in the {rhos_acro} control plane:
 +
 ----
-openstack subnet set --dns-nameserver 192.168.122.80 provisioning-subnet
+$ openstack subnet set --dns-nameserver 192.168.122.80 provisioning-subnet
 ----
 
-. Your {bare_metal} nodes might be missing from a `openstack baremetal node list` command due to increased access restrictions in the role-based access control model. To see the nodes again, temporarily disable the new role- based access control policy, which you can then re-enable after setting the 'owner' field on the nodes.
+. If your {bare_metal} nodes are missing from the `openstack baremetal node list` command, temporarily disable the new RBAC policy to see the nodes again. You can then re-enable RBAC after setting the `owner` field on the nodes:
```

Comment:
Before the step about the nodes missing, should we have a step that tells users to run the `openstack baremetal node list` command, and then turn this step into a note that tells them what to do if the nodes are missing?

Reply:
1. Yes, a step to run `openstack baremetal node list` to verify first makes sense.
2. No, setting the owner field does not re-enable RBAC, the steps are
  1. Disable RBAC
  2. Set `owner` field
  3. Re-enable RBAC

Reply:
@hjensas It looks like the code in this step is only showing customers how to re-enable RBAC (see below). I think we need two steps before that to show 1) how to temporarily disable RBAC and 2) how to set the owner field. If you agree, can you please provide those steps in a comment on this PR?
```
----
$ oc patch openstackcontrolplane openstack -n openstack --type=merge --patch '
spec:
  ironic:
    enabled: true
    template:
      databaseInstance: openstack
      ironicAPI:
        replicas: 1
        customServiceConfig: |
          [oslo_policy]
          enforce_scope=false
          enforce_new_defaults=false
'
----
```

Reply:
@hjensas Can you please see my question above? ^

Reply:
These settings disable RBAC:
```
          [oslo_policy]
          enforce_scope=false
          enforce_new_defaults=false
```

To re-enable it, either change the value to `true` - or simply delete the lines from `customServiceConfig`.